<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window2</class>
 <widget class="QMainWindow" name="Window2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>521</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-1822</y>
       <width>462</width>
       <height>2341</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="17" column="1">
       <widget class="QPushButton" name="pushButton_6">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
	color:white;
    min-width: 10em;
    padding: 6px;</string>
        </property>
        <property name="text">
         <string>Hanya Rp.75.499</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="3">
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
color:black;
font-size:20pt;
</string>
        </property>
        <property name="text">
         <string>Dari penulis Tere Liye</string>
        </property>
       </widget>
      </item>
      <item row="29" column="3">
       <widget class="QPushButton" name="pushButton_11">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
	color:white;
    min-width: 10em;
    padding: 6px;</string>
        </property>
        <property name="text">
         <string>Hanya Rp.68.799</string>
        </property>
       </widget>
      </item>
      <item row="27" column="1">
       <widget class="QLabel" name="judul_dilan1">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>BUKU</string>
        </property>
       </widget>
      </item>
      <item row="25" column="1" colspan="3">
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
color:black;
font-size:20pt;
</string>
        </property>
        <property name="text">
         <string>Kisah Dilan dan Milea</string>
        </property>
       </widget>
      </item>
      <item row="19" column="1" colspan="3">
       <widget class="QLabel" name="label_15">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
color:black;
font-size:20pt;
</string>
        </property>
        <property name="text">
         <string>Kisah hidup orang hebat</string>
        </property>
       </widget>
      </item>
      <item row="18" column="1">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="26" column="1">
       <widget class="QLabel" name="label_17">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/img/img/buku6.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="12" column="3">
       <widget class="QPushButton" name="pushButton_4">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
	color:white;
    min-width: 10em;
    padding: 6px;</string>
        </property>
        <property name="text">
         <string>Hanya Rp.75.499</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QLabel" name="judul_tere3">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>BUKU</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="24" column="1">
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="22" column="1">
       <widget class="QLabel" name="penulisbio1">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Penulis</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="23" column="1">
       <widget class="QPushButton" name="pushButton_8">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
	color:white;
    min-width: 10em;
    padding: 6px;</string>
        </property>
        <property name="text">
         <string>Hanya Rp.89.999</string>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/img/img/buku10.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="judul_populer1">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>BUKU</string>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QLabel" name="judul_tere2">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>BUKU</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="13" column="3">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="penulis_populer2">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Penulis</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="21" column="1">
       <widget class="QLabel" name="judulbio1">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>BUKU</string>
        </property>
       </widget>
      </item>
      <item row="15" column="3">
       <widget class="QLabel" name="judul_tere4">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>BUKU</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="pushButton_3">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
	color:white;
    min-width: 10em;
    padding: 6px;</string>
        </property>
        <property name="text">
         <string>Rp.89.999</string>
        </property>
       </widget>
      </item>
      <item row="22" column="3">
       <widget class="QLabel" name="penulisbio2">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Penulis</string>
        </property>
       </widget>
      </item>
      <item row="16" column="3">
       <widget class="QLabel" name="penulistere4">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Penulis</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
color:black;
font-size:20pt;
</string>
        </property>
        <property name="text">
         <string>Buku Populer</string>
        </property>
       </widget>
      </item>
      <item row="14" column="3">
       <widget class="QLabel" name="label_11">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/img/img/buku8.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="28" column="1">
       <widget class="QLabel" name="penulisdilan1">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Penulis</string>
        </property>
       </widget>
      </item>
      <item row="20" column="1">
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/img/img/buku1.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="17" column="3">
       <widget class="QPushButton" name="pushButton_7">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
	color:white;
    min-width: 10em;
    padding: 6px;</string>
        </property>
        <property name="text">
         <string>Hanya Rp.75.499</string>
        </property>
       </widget>
      </item>
      <item row="23" column="3">
       <widget class="QPushButton" name="pushButton_9">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
	color:white;
    min-width: 10em;
    padding: 6px;</string>
        </property>
        <property name="text">
         <string>Hanya Rp.69.999</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/img/img/buku9.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLabel" name="penulistere1">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Penulis</string>
        </property>
       </widget>
      </item>
      <item row="26" column="3">
       <widget class="QLabel" name="label_18">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/img/img/buku7.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLabel" name="judul_tere1">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>BUKU</string>
        </property>
       </widget>
      </item>
      <item row="11" column="3">
       <widget class="QLabel" name="penulistere2">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Penulis</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="16" column="1">
       <widget class="QLabel" name="penulistere3">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Penulis</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QPushButton" name="pushButton_5">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
	color:white;
    min-width: 10em;
    padding: 6px;</string>
        </property>
        <property name="text">
         <string>Hanya Rp.75.499</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="penulis_populer1">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Penulis</string>
        </property>
       </widget>
      </item>
      <item row="28" column="3">
       <widget class="QLabel" name="penulisdilan2">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Penulis</string>
        </property>
       </widget>
      </item>
      <item row="29" column="1">
       <widget class="QPushButton" name="pushButton_10">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
	color:white;
    min-width: 10em;
    padding: 6px;</string>
        </property>
        <property name="text">
         <string>Hanya Rp.68.799</string>
        </property>
       </widget>
      </item>
      <item row="27" column="3">
       <widget class="QLabel" name="judul_dilan2">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>BUKU</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="18" column="3">
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: orange;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
	color:white;
    min-width: 10em;
    padding: 6px;</string>
        </property>
        <property name="text">
         <string>Rp. 68.799</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/img/img/buku4.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/img/img/buku6.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="20" column="3">
       <widget class="QLabel" name="label_13">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/img/img/buku5.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/img/img/buku1.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="21" column="3">
       <widget class="QLabel" name="judulbio2">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>BUKU</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="judul_populer2">
        <property name="font">
         <font>
          <family>Goldplay</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>BUKU</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
